Softeq.XToolkit.WhiteLabel.Essentials.Droid/ImagePicker/IImagePickerActivityResultHandler.cs

```diff
@@ -0,0 +1,16 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+using Android.Graphics;
+
+namespace Softeq.XToolkit.WhiteLabel.Essentials.Droid.ImagePicker
+{
+    public interface IImagePickerActivityResultHandler
```

Comment:
Looking on how this interface is going to be used, here is the idea of how it might be enhanced. We need to handle two types of requests + we might need a way to extend logic for new types of requests, so we could implement three separate methods here:
- HandleImagePickerCameraResultAsync(Result resultCode, Intent data, Uri fileUri);
- HandleImagePickerGalleryResultAsync(Result resultCode, Intent data);
- HandleCustomResultAsync(int requestCode, Result resultCode, Intent data);